### ISSUE TITLE
hi,do you have wolapi for ra2 1006 ?

### ISSUE THREAD

**Issue by showname2020** [hifi-unmaintained/wolapi#1]
hi,do you have wolapi for ra2 1006 ?

**Comment by OmniBlade** [comment]
No, this reimplementation only support RA 3.xx as far as I know.

**Comment by hifi** [comment]
I may have tested this with TS or RA2 but can't remember anymore.

It may work or it may need some work.

**Comment by showname2020** [comment]
Do you have any other versions of wolapi?

**Comment by showname2020** [comment]
Or an unmodified version of wolapi，thank you

**Comment by showname2020** [comment]
/join #cncnet
#cncnet: Cannot join channel (+k) - bad key
